python/nano/src/bigdl/nano/pytorch/onnx/onnxrt_inference.py

```diff
@@ -188,6 +188,11 @@ def on_fit_start(self):
 
 
 def _forward_onnx(self, *args):
+    # if self.train() is called without calling self.exit_onnx()
+    if self.training:
+        self.exit_onnx()
```

Comment:
shall we set `self._ortsess_up_to_date = False` here?

Reply:
yes, and since we need to support pytorch users other than pl users, there are many details to be considered. I will switch this PR to WIP for now.

Reply:
> yes, and since we need to support pytorch users other than pl users, there are many details to be considered. I will switch this PR to WIP for now.

I think you should just override the `train` method?